<ui version="4.0" >
 <class>SetupInterface</class>
 <widget class="QWidget" name="SetupInterface" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Setup Interface</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>511</width>
     <height>202</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <property name="horizontalSpacing" >
     <number>12</number>
    </property>
    <property name="verticalSpacing" >
     <number>12</number>
    </property>
    <item row="0" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>wireless interface</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLineEdit" name="leWInterface" >
      <property name="text" >
       <string>ath0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>wireless driver</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QLineEdit" name="leWDriver" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>madwifi</string>
      </property>
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>wired interface</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QLineEdit" name="leInterface2" >
      <property name="text" >
       <string>eth0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" >
     <widget class="QCheckBox" name="cbNFCInterface" >
      <property name="text" >
       <string>NFC interface</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1" >
     <widget class="QLineEdit" name="leNFCInterface" >
      <property name="text" >
       <string>/dev/ttyUSB0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" >
     <widget class="QCheckBox" name="cbIPAddress" >
      <property name="text" >
       <string>IP address</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1" >
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>12</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="leIPAddress" >
        <property name="text" >
         <string>192.168.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>net mask</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leNetMask" >
        <property name="text" >
         <string>255.255.255.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="label_5" >
      <property name="text" >
       <string>bridge interface</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" >
     <widget class="QLineEdit" name="leInterface3" >
      <property name="text" >
       <string>br0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leWInterface</tabstop>
  <tabstop>leWDriver</tabstop>
  <tabstop>leInterface2</tabstop>
  <tabstop>leInterface3</tabstop>
  <tabstop>cbIPAddress</tabstop>
  <tabstop>leIPAddress</tabstop>
  <tabstop>leNetMask</tabstop>
  <tabstop>cbNFCInterface</tabstop>
  <tabstop>leNFCInterface</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
